libraries/AP_XRCE_Client/wscript

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+def configure(cfg):
+    cfg.env.XRCE_DEPENDENCIES = []
+    cfg.env.XRCE_DEPENDENCIES+=[str(source) for source in (cfg.srcnode.ant_glob('modules/Micro-XRCE-DDS-Client/src/c/core/session/stream/*.c'))]
```

Comment:
Why does ardupilot care about the specific source structure of Micro-XRCE-DDS-Client? Can we have Waf connected to their CMakeLists and just link to their build libraries or cmake targets? 

If EProsima re-organizes folder structure, it breaks our build. 
If they change implementation file extensions to cpp, it breaks out build. 
If they add more files in another folder that need to be compiled but are private implementations, it breaks our build.